### PR TITLE
Insert redux and redux-saga in the mix

### PR DESCRIPTION
**Pull request by renanrodrigo**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3345,24 +3345,6 @@
...
-      },
-      "dependencies": {
-        "regenerator-runtime": {
-          "version": "0.11.1",
-          "resolved": "https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.11.1.tgz",
-          "integrity": "sha512-MguG95oij0fC3QV3URf4V2SDYGJhJnJGqvIIgdECeODCT98wSWDAJ94SSuVpYQUoTcGUIL6L4yNB7j1DFFHSBg==",
-          "dev": true
-        }
-      }
-    },
```
Comment (StykMartin) [inline package-lock.json]: This was added in previous commit.
https://github.com/beaker-project/beaker-dashboard/pull/57/commits/f7057a16f659c5d0681d8efb63115c929acaeeef

Rebase and drop it.

File: package.json [hunk 2]
```diff
@@ -57,7 +57,6 @@
     "raw-loader": "^4.0.1",
     "redux-mock-store": "^1.5.4",
     "redux-saga-test-plan": "^4.0.0-rc.3",
-    "redux-saga-tester": "^1.0.801",
```
Comment (StykMartin) [inline package.json]: Added in https://github.com/beaker-project/beaker-dashboard/pull/57/commits/6656d2cc683eade1eea799fff70ddfa2aa74e946

Rebase.

Comment (StykMartin) [inline package.json]: Added in 6656d2c

Rebase.

**Comment by renanrodrigo** [comment]
post-fixes rebased

**Comment by StykMartin** [comment]
Well done @renanrodrigo,

Just fix the conflicts and we good to go.

**Comment by renanrodrigo** [comment]
Thanks, @StykMartin , will do.

**Comment by renanrodrigo** [comment]
Rebased and fixed

**Pull request merged**